src/components/dropdown/__spec__.js

```diff
@@ -925,18 +925,6 @@ describe('Dropdown', () => {
         expect(instance.additionalInputContent.length).toEqual(0);
       });
     });
-
-    it('creates the list in a Portal', () => {
-      const wrapper = mount(
-        <Dropdown
-          name='foo'
-          options={ Immutable.fromJS([{ id: 1, name: 'foo' }, { id: 2, name: 'bar' }]) } value='1'
-        />);
-
-      wrapper.find('.carbon-dropdown__input').simulate('focus');
-      const portalDropdownList = wrapper.find(Portal).find('.carbon-dropdown__list');
-      expect(portalDropdownList).toMatchSnapshot();
-    });
```

Comment:
Similarly, I'm not sure why this spec has been removed?

Reply:
Yes, that's my mistake. When I was removing the Portal `guid` code and specs I removed too much. I will re-add. That's why the snapshot is missing too.